rbc/tests/heavydb/test_text_encoding_dict.py

```diff
@@ -488,3 +494,26 @@ def test_getString_lst(heavydb, col):
         assert list(result) == [(9,), (9,), (9,), (12,), (15,)]
     else:
         assert list(result) == [(18,), (9,), (9,), (9,)]
+
+
+@pytest.mark.parametrize('col', ['t1'])
+def test_udf_copy_dict_encoded_string(heavydb, col):
+    if heavydb.version[:2] < (6, 2):
+        pytest.skip('Requires HeavyDB main branch')
+
+    table = f"{heavydb.base_name}text"
+    query = f"SELECT fn_copy({col}) from {table}"
+    _, result = heavydb.sql_execute(query)
+
+    assert list(result) == [('fun',), ('bar',), ('foo',), ('barr',), ('foooo',)]
+
+
+def test_row_function_manager(heavydb):
+    if heavydb.version[:2] < (6, 3):
+        pytest.skip('Requires HeavyDB main branch')
+
+    @heavydb('int32(int32, RowFunctionManager)')
+    def invalid_fn(a, mgr):
+        return a
```

Comment:
Did you mean adding this within a try-except block?

Reply:
Yes! Fixed